src/postgres/migrations/0009_delete_ext2020.py

```diff
@@ -0,0 +1,19 @@
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('reviews', '0001_initial'),
+        ('postgres', '0008_fix_user_id_group'),
+    ]
+
+    operations = [
+        migrations.RunSQL('''
+        drop table if exists ext2020;
+        delete from auth_permission where content_type_id in (select id from django_content_type where app_label = '{app_label}');
+        delete from django_admin_log where content_type_id in (select id from django_content_type where app_label = '{app_label}');
+        delete from django_content_type where app_label = '{app_label}';
+        delete from django_migrations where app='{app_label}';
+        '''.format(app_label='ext2020'))
```

Comment:
I think using `./manage.py migrate ext2020 zero` before this PR merged is better than use SQL to delete the table.
ref: https://stackoverflow.com/questions/3329773/django-how-to-completely-uninstall-a-django-app

Reply:
Hi @setmao 

I  have used this command in my local environment. 
But I don't how to do this command in product environment

Reply:
I'll handle the deployment so don't worry about staging/production.

Reply:
@kaka-lin I think we can delete this file and let @mattwang44 handle the migration in staging/production.

Reply:
@setmao OK